### PR TITLE
[Wallet][ReDesign] Language selection

### PR DESCRIPTION
**Pull request by jeanregisser**
### Description

This implements the [language selection redesign](https://www.figma.com/file/zt7aTQ5wuXycIwxq5oAmF9/Wallet--Refresh?node-id=4962%3A415).

### Related issues

- Epic #3645

### Backwards compatibility

Yes.

### CODE REVIEW

File: packages/mobile/src/language/Language.tsx [hunk 1]
```diff
@@ -1,51 +1,63 @@
...
-  const route = useRoute<RouteProp<StackParamList, Screens.Language>>()
+type Props = StackScreenProps<StackParamList, Screens.Language>
+
+export default function Language({ navigation, route }: Props) {
   const dispatch = useDispatch()
-  const { t } = useTranslation(Namespaces.accountScreen10)
+  const { t, i18n } = useTranslation(Namespaces.accountScreen10)
 
-  const onSelectAnswer = (language: string, code: string) => {
+  const onSelect = (language: string, code: string) => {
```
Comment (i1skn) [inline packages/mobile/src/language/Language.tsx]: nit: we can define it as `function onSelect(...) { ... }`

**Review by i1skn — COMMENTED** [review]
Overall great work! One thing I've noticed - if this screen accessed from Settings - there is no back button in the Header. Do you think we can add as a part of this PR?

**Comment by jeanregisser** [comment]
> Overall great work! One thing I've noticed - if this screen accessed from Settings - there is no back button in the Header. Do you think we can add as a part of this PR?

True, I'm leaving it for when we update the settings.

**Review by jmrossy — APPROVED** [review]
LGTM! Nice work!

**Pull request merged**